src/components/schemas/report.yaml

```diff
@@ -159,6 +159,179 @@ report_out:
+    purpose:
+      type: string
+      description: purpose for creating the report
+      example: Business trip to London
+      nullable: True
+    currency:
+      $ref: ./fields.yaml#/currency
+    amount:
+      $ref: ./fields.yaml#/amount
+    tax:
+      type: number
+      description: |
+        Tax value of the object.
+      example: 18.23
+      nullable: True
+    state:
+      type: string
+      enum:
+        - DRAFT
+        - APPROVER_PENDING
+        - APPROVER_INQUIRY
+        - PAYMENT_PROCESSING
+        - APPROVED
+        - PAYMENT_PENDING
+        - PAID
+    num_expenses:
+      type: integer
+      description: |
+        Number of expenses part of this report
+      example: 3
+    is_verified:
+      type: boolean
+      nullable: true
+      description: |
+        Determines whether this report has been verified by a verifier
+    settlement_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_string'
+      nullable: true
+      example: 'set5nlgnwgwl'
+    is_physical_bill_submitted:
+      type: boolean
+      nullable: true
+      description: |
+        If physical has been collected for this report
+    physical_bill_submitted_at:
+      type: string
+      format: date-time
+      example: '2020-06-11T13:14:55.201598+00:00'
+      description: |
+        Signifies if and when the physical bill for this report was submitted.
+      nullable: true
+    is_manually_flagged:
+      type: boolean
+      description: |
+        Determines whether this report was flagged manually
+    is_policy_flagged:
+      type: boolean
+      description: |
+        Determines whether this report was flagged from a policy
+    is_exported:
+      type: boolean
+      nullable: true
+      description: |
+        Determines whether this expense has been exported to external accounting system or not.
+    last_paid_at:
+      type: string
+      format: date-time
+      example: '2020-06-11T13:14:55.201598+00:00'
+      description: |
+        Signifies if and when this report was reimbursed.
+      nullable: True
+    last_approved_at:
+      type: string
+      format: date-time
+      example: '2020-06-11T13:14:55.201598+00:00'
+      description: |
+        Signifies if and when this report was approved.
+      nullable: True
+    last_submitted_at:
+      type: string
+      format: date-time
+      example: '2020-06-11T13:14:55.201598+00:00'
+      description: |
+        Signifies if and when this report was submitted.
+      nullable: True
+    last_resubmitted_at:
+      type: string
+      format: date-time
+      example: '2020-06-11T13:14:55.201598+00:00'
+      description: |
+        Signifies if and when this report was resubmitted.
+      nullable: True
+    seq_num:
+      type: string
+      description: |
+        A unique human readable identifier for the report
+      example: 'C/2021/02/R/907'
+    source:
+      type: string
+      description: |
+        where was this report object created from
```

Comment:
signifes the client from which this report was created, also should be enum no ?